microsite/data/plugins/changelog.yaml

```diff
@@ -0,0 +1,12 @@
+---
+title: Changelog viewer
+author: RSC Labs
+authorUrl: https://rsoftcon.com/
+category: Discovery
+description: View changelogs for your components in Backstage.
+documentation: https://github.com/RSC-Labs/backstage-changelog-plugin/blob/main/README.md
+iconUrl: https://raw.githubusercontent.com/RSC-Labs/backstage-changelog-plugin/main/docs/plugin_icon.png
+npmPackageName: '@rsc-labs/changelog-plugin'
```

Comment:
Have you published this yet? I can't find it searching and this URL should work but doesn't: https://www.npmjs.com/package/@rsc-labs/changelog-plugin

Reply:
Yes, sorry, misspelling - I will correct it. The correct package name is @rsc-labs/backstage-changelog-plugin